scripts/tests/chiptest/test_definition.py

```diff
@@ -264,7 +266,7 @@ def items_with_key(self):
             (
                 # This path varies, however it is a fixed python tool so it may be ok
                 self.chip_tool_with_python_cmd,
-                os.path.basename(self.chip_tool_with_python_cmd[-1]),
+                os.path.basename(self.chip_tool_with_python_cmd.args[-1]),
```

Comment:
I think that `self.chip_tool_with_python_path.name` should be here. Please double check.